simple-singolo/style.css

```
@@ -0,0 +1,1017 @@
+  }
+  50% {
+    transform: translateX(50vw);
+  }
+  100% {
+    transform: translateX(0vw);
+  }
+}
+.element4 {
+  animation-name: pulse4;
+  animation-duration: 5s;
+  animation-timing-function: ease-out;
+  animation-delay: 0s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-play-state: running;
+}
+
+.display {
+  display: none;
+}
+
+html {
+  font-size: 10px;
+  scroll-behavior: smooth;
+}
+* {
+  margin: 0;
+}
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.hamburger-wrapper {
+  display: none;
+}
+.rotate {
+  transform: rotate(90deg);
+}
+.hamburger {
+  width: 25px;
+  height: 20px;
+  position: relative;
+}
+.hamburger_line {
+  display: inline-block;
+  background: var(--wite);
+  border-radius: 5px;
+  height: 2.4px;
+  width: 100%;
+  position: absolute;
+  left: 0px;
+  top: calc(50% - 1.5px);
+}
+.hamburger::before {
+  content: "";
+  display: inline-block;
+  background: var(--wite);
+  border-radius: 5px;
+  height: 3px;
+  width: 100%;
+  position: absolute;
+  left: 0px;
+  top: 0;
+}
+.hamburger::after {
+  content: "";
+  display: inline-block;
+  background: var(--wite);
+  border-radius: 5px;
+  height: 3px;
+  width: 100%;
+  position: absolute;
+  left: 0px;
+  bottom: 0px;
+}
+.h2 {
+  font-family: "Lato900";
+  font-size: 3rem;
+  line-height: 3.6rem;
+  text-transform: capitalize;
+  color: var(--section-headers);
+  letter-spacing: -0.006em;
+}
+.information {
+  font-family: "Lato300";
+  font-size: 1.8rem;
+  line-height: 3rem;
+  color: var(--simple-text);
+}
+.wrapper {
+  max-width: 1020px;
+  min-width: 320px;
+}
+
+/* Header */
+.header {
+  width: 100vw;
+  background-color: var(--dark-background);
+  border-bottom: solid 6px var(--bprder-dark-color);
+  display: flex;
+  justify-content: center;
+}
+.header__wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 0 29px;
+  width: 940px;
+}
+.logo {
+  position: relative;
+  font-family: "LatoBold";
+  font-size: 2.5rem;
+  line-height: 3rem;
+  letter-spacing: -0.03em;
+  text-transform: uppercase;
+}
+.logo__text {
+  color: var(--wite);
+}
+.logo_star {
+  position: absolute;
+  top: 0;
+  right: top;
+  color: var(--active-item-navigation);
+}
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  font-family: "LatoBold";
+  padding-top: 11px;
+  width: 375px;
+  font-size: 1.2rem;
+  line-height: 1.4rem;
+  letter-spacing: -0.03em;
+  text-transform: uppercase;
+  color: var(--wite);
+}
+.navigation > li {
+  list-style: none;
+}
+.navigation__item:not(.navigation__active-item):hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+.navigation__item {
+  text-decoration: none;
+  color: var(--wite);
+  position: relative;
+}
+.navigation__item:before {
+  content: ".";
+  position: absolute;
+  top: -0.35rem;
+  left: -1.75rem;
+}
+.navigation__first-item:before {
+  content: " ";
+}
+.navigation__active-item {
```

```suggestion
.navigation__item--active {
```
this is definitely a bem modifier